### ISSUE TITLE
Remove interior faces at AMR level boundaries.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#2126]
I tried looking for keywords like amr, ghost, interior, bound to see if there was an issue for this but I did not find anything.

When you have a 3D AMR mesh, VisIt does not remove all faces shared between patches at level boundaries. It looks like some amount of removal is done but much of the boundaries remain. This causes the plots to look bad when transparency is used (transparent Subset, transparent Pseudocolor).

This happens for a variety of AMR file formats and it suggests that VisIt's ghost zone generation is not quite producing enough ghost data to remove the interior faces at AMR level boundaries.

It may be that something like the AMRStitchCell operator eventually gets improved enough to be moved into VisIt's ghost zone generation routines.

*Done:*
This issue is done when a set of different 3D rectilinear AMR datasets can be plotted using the Subset and Pseudocolor plots with transparency and have no inter-level interior geometry that fouls the look of the transparent plot (when looking at the entire dataset). VisIt must handle this automatically, with no special operators, etc. What is desired in that case is a transparent box that would match what would be plotted as when a single-domain rectilinear grid is plotted using transparency(see attached image that provides a mock-up). This will work in parallel too.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 06/26/2014 02:54 pm
Original update: 07/08/2014 04:03 pm
Ticket number: 1893
